OpenRA.Mods.Common/Traits/BotModules/UnitBuilderBotModule.cs

```diff
@@ -101,36 +104,19 @@ int IBotRequestUnitProduction.RequestedProductionCount(IBot bot, string requeste
 			return queuedBuildRequests.Count(r => r == requestedActor);
 		}
 
-		void BuildUnit(IBot bot, string category, bool buildRandom)
+		void BuildUnitRandomly(IBot bot, string category)
```

Comment:
`BuildRandomUnit` is clearer.

Reply:
why no longer `buildRandom`? because it was always set to `true`?

Reply:
Because I really don't see how `buildRandom` makes the differences: `ChooseRandomUnitToBuild` build unit randomlly and `ChooseUnitToBuild` also build unit randomly but consider the unit fraction. I think it is better that we simply use the way from `ChooseUnitToBuild` for better unit fraction control.

Reply:
> BuildRandomUnit is clearer.

fixed